src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -124,66 +99,88 @@ function getRouteForCurrentStep(currentStep: TBankAccountStep): ValueOf<typeof R
     }
 }
 
-function ReimbursementAccountPage({
-    reimbursementAccount,
-    route,
-    onfidoToken = '',
-    policy,
-    account,
-    isLoadingApp = false,
-    session,
-    plaidLinkToken = '',
-    plaidCurrentEvent = '',
-}: ReimbursementAccountPageProps) {
+/**
+ * Returns selected bank account fields based on field names provided.
+ */
+function getFieldsForStep(step: TBankAccountStep): InputID[] {
+    switch (step) {
+        case CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT:
+            return ['routingNumber', 'accountNumber', 'bankName', 'plaidAccountID', 'plaidAccessToken', 'isSavings'];
+        case CONST.BANK_ACCOUNT.STEP.COMPANY:
+            return [
+                'companyName',
+                'addressStreet',
+                'addressZipCode',
+                'addressCity',
+                'addressState',
+                'companyPhone',
+                'website',
+                'companyTaxID',
+                'incorporationType',
+                'incorporationDate',
+                'incorporationState',
+            ];
+        case CONST.BANK_ACCOUNT.STEP.REQUESTOR:
+            return ['firstName', 'lastName', 'dob', 'ssnLast4', 'requestorAddressStreet', 'requestorAddressCity', 'requestorAddressState', 'requestorAddressZipCode'];
+        default:
+            return [];
+    }
+}
+
+function ReimbursementAccountPage({route, policy}: ReimbursementAccountPageProps) {
+    const session = useSession();
+    const [isLoadingApp] = useOnyx(ONYXKEYS.IS_LOADING_APP);
```

Comment:
Should we add a fallback value as before?

Same to plaidLinkToken, onfidoToken, plaidCurrentEvent

Reply:
I don't think we need it

Reply:
@koko57 Why don't you think we need it? Let's see the our previous implementation

<img width="453" alt="Screenshot 2024-10-01 at 13 50 18" src="https://github.com/user-attachments/assets/44c7d16d-1f9b-4905-bf8e-0e949b0ba69e">

Reply:
@DylanDylann I don't see anywhere in the app we're using fallback values for the values from useOnyx. And here if we get undefined or null to won't break anything, the conditions will work the same and the TS is not complaining. So I don't see the reason to add these fallbacks.

Reply:
but ok, I see that it was implemented in the conflicting file